tests/__tests__/mixins/labels.js

```diff
@@ -32,15 +51,4 @@ describe("Atom labels", () => {
         wave.toggleLabels();
         expect(labels.every((label) => label.visible === !areLabelsInitiallyShown)).toBeTruthy();
     });
-
-    test("Label center is positioned on the atom surface", () => {
```

Comment:
Seems like a good thing to test for - Label center is positioned on the atom surface - perhaps we should keep at least the explanation, if you are doing it in the above test